docs/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I wonder if this is going to change, knowing that AI is using this stuff to do people's work